internal/provider/provider_test.go

```diff
@@ -71,6 +84,14 @@ func SkipJAAS(t *testing.T) {
 	}
 }
 
+// OnlyCrossController should be called at the top of any tests that are
+// specific to cross-controller functionality.
+func OnlyCrossController(t *testing.T) {
```

Comment:
One more name suggestion to throw on top of my previous ones,
`NeedsCrossController`, take whichever you prefer.